recipe/meta.yaml

```diff
@@ -1,53 +1,58 @@
 {% set name = "cattrs" %}
-{% set version = "22.1.0" %}
-{% set sha256 = "94b67b64cf92c994f8784c40c082177dc916e0489a73a9a36b24eb18a9db40c6" %}
+{% set version = "23.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/python-attrs/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9dbc7359c1ac2acdd011607218fd2aac26aab5fdaa51a7e162110d029e277ff3
 
 build:
   number: 0
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-    - poetry-core >=1.0.0
-    - wheel
+    - poetry-core >=1.1.0
   run:
     - python
-    - attrs >=20.1.0
-    - exceptiongroup  # [py<311]
-    - typing_extensions  # [py<38]
+    - attrs >=20
+    - exceptiongroup             # [py<311]
+    - typing_extensions >=4.1.0  # [py<311]
+  run_constrained:
+    - ujson >=5.4.0,<6
```

Comment:
These are overly strict. I'd go as far as saying completely unnecessarily strict and will just cause pain and conflicts all over the place. These strict requirements are caused by poetry's way of doing things similarly to how things are done in JS.

Even if we go with keeping only the lower bounds, it'll probably cause issues to users. So I'd suggest to just not set any `run_constrained`.

Reply:
ok, removed